service/service.go

```diff
@@ -73,6 +74,21 @@ func New(config Config) (*Service, error) {
 
 	var err error
 
+	resourceGroup := config.Viper.GetString(config.Flag.Service.Azure.HostCluster.ResourceGroup)
+	if resourceGroup == "" {
+		resourceGroup = config.Viper.GetString(config.Flag.Service.Installation.Name)
+	}
+
+	virtualNetwork := config.Viper.GetString(config.Flag.Service.Azure.HostCluster.VirtualNetwork)
+	if virtualNetwork == "" {
+		virtualNetwork = resourceGroup
+	}
+
+	virtualNetworkGateway := config.Viper.GetString(config.Flag.Service.Azure.HostCluster.VirtualNetworkGateway)
+	if virtualNetworkGateway == "" {
+		virtualNetworkGateway = fmt.Sprintf("%s-%s", resourceGroup, "vpn-gateway")
+	}
+
```

Comment:
In case the resource group parameter is empty, we default it to the installation name.
The virtual network resource in that resource group is also named after the installation name if I'm not mistaken.

The virtual network gateway is called ${installation_name}-vpn-gateway, so we default to that.